sos/plugins/unpackaged.py

```diff
@@ -0,0 +1,86 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+import re
+
+
+class Unpackaged(Plugin, RedHatPlugin):
+    '''
+    Collects a list of files that are not handled by the package
+    manager
+    '''
+
+    def setup(self):
+
+        def all_files_rpm():
+            """
+            It returns a list of files known by the package manager
+            """
+            return self.get_command_output('rpm -qal')['output'].splitlines()
+
+        def get_env_path_list():
+            """
+            It returns a list of directories contained in the $PATH var
+            """
+            return os.environ['PATH'].split(':')
+
+        def check_usrmove():
```

Comment:
Nack: this is really a `Policy` item, not `Plugin`. Plugins should either call into the policy or this should be abstracted so that they don't need to care about it.